src/NovelRT.Interop/CMakeLists.txt

```diff
@@ -58,41 +58,34 @@ target_include_directories(Interop
     $<INSTALL_INTERFACE:include>
 )
 
+target_compile_options(Interop
+  PRIVATE
+    $<$<CXX_COMPILER_ID:MSVC>:/W4>
+    $<$<CXX_COMPILER_ID:MSVC>:/WX>
+    $<$<CXX_COMPILER_ID:MSVC>:/wd4611>
+    $<$<CXX_COMPILER_ID:MSVC>:/MP>
+    $<$<CXX_COMPILER_ID:GNU>:-pedantic>
+    $<$<CXX_COMPILER_ID:GNU>:-pedantic-errors>
+    $<$<CXX_COMPILER_ID:GNU>:-Wall>
+    $<$<CXX_COMPILER_ID:GNU>:-Wextra>
+    $<$<CXX_COMPILER_ID:GNU>:-Werror>
+    $<$<CXX_COMPILER_ID:GNU>:-Wno-float-equal>
+    $<$<CXX_COMPILER_ID:GNU>:-Wno-padded>
+    $<$<CXX_COMPILER_ID:Clang>:-pedantic>
```

Comment:
We probably also want to include duplicates of these for AppleClang, as CMake detects Apple's distribution as a different compiler. Personally, I'd also have a newline between each group too.

Reply:
Done